stable/toxiproxy/README.md

```diff
@@ -0,0 +1 @@
+TODO
```

Comment:
This is autogenerated like this: `docker run --rm -v "$PWD:/helm-docs" jnorwood/helm-docs:latest --template-file=../../ci/README.md.gotmpl`

Details here: https://github.com/deliveryhero/helm-charts#contributing